### PR TITLE
🗺️ Added /archive pages to sitemap

### LINKED ISSUE #2698: 🚶 Sitemap - reference archive sitemaps in main sitemap index
Once all the old pages are archived, and redirects are in place - we need to include the archive sitemaps in our main sitemap file.

### PR DESCRIPTION
**Pull request by Calinator444**
### Description
I'm adding the /archive sitemap to the list of pages for Google to crawl by adding it to our additionalSitemaps.

**Note**: This PR should not be merged prior to merging this one: https://github.com/SSWConsulting/SSW.Website-v1-Progress/pull/135

### Solves Issue

Fixed #2698

### CODE REVIEW

File: next-sitemap.config.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ module.exports = {
     additionalSitemaps: [
       "https://www.ssw.com.au/people/sitemap-index.xml",
       "https://www.ssw.com.au/rules/sitemap-index.xml",
+      "https://www.ssw.com.au/archive/sitemap.xml",
```
Comment (amankumarrr) [inline next-sitemap.config.js]: Can we follow the same format? (i.e */archive/sitemap-index.xml)

**Review by amankumarrr — CHANGES_REQUESTED** [review]

File: next-sitemap.config.js (hunk 1 above)
Comment (Calinator444) [reply next-sitemap.config.js]: Sure, just let me update my other PR

Comment (Calinator444) [reply next-sitemap.config.js]: Done - I'll need my PR approved first though
https://github.com/SSWConsulting/SSW.Website-v1-Progress/pull/135

Comment (amankumarrr) [reply next-sitemap.config.js]: Approved that PR.

Comment (Calinator444) [reply next-sitemap.config.js]: Done - I moved https://www.ssw.com.au/archive/sitemap.xml to https://www.ssw.com.au/archive/sitemap-index.xml

**Review by amankumarrr — APPROVED** [review]

**Pull request merged**